Makefile

```diff
@@ -7,12 +7,18 @@ SKOPEO_BRANCH = master
 SUDO =
 
 ifeq ($(shell uname),Darwin)
-DARWIN_BUILD_TAG = containers_image_ostree_stub
+PLATFORM_BUILD_TAG = containers_image_ostree_stub containers_image_openpgp
```

Comment:
No, non-OpenPGP builds are expected to work natively.  Maybe you want to define a separate cross-build flags variable instead?

Reply:
only when compiling on a darwin host, it can link to gpgme. But if you're cross compiling for darwin then it can't do the gpgme linking

Reply:
Right, but this is the `shell uname` condition, when we _are_ on a Darwin host.

Reply:
look at how i've revised it